### ISSUE TITLE
Duplicate dock after suspension 

### ISSUE THREAD

**Issue by giovannilaperna** [micheleg/dash-to-dock#1731]
This is very similar to [issue 1727](https://github.com/micheleg/dash-to-dock/issues/1727) but different.

On Ubuntu 20.04, only after suspension, when clicking on 'Show Applications' a duplicate dock is showing.

I'm not sure if this is the original Ubuntu dock or the dash-to-dock without custom settings (I guess is the latter).

See screenshot:

![Screenshot 2022-05-04 20:36:36](https://user-images.githubusercontent.com/5690126/166848926-a4cef736-907b-4ee9-a731-173849b7fab7.png)

**Comment by emansom** [comment]
Duplicate of #1700. Ubuntu ships a fork of dash to dock that's enabled by default.

**Comment by emansom** [comment]
@3v1n0 @micheleg @vanvugt Given this issue has been raised several times and the default behavior is confusing newcomers, might it be a option to built a presence detection of some sort within Ubuntu Dock that it disables itself when the regular Dash to Dock extension is enabled?

**Comment by giovannilaperna** [comment]
I disabled Ubuntu Dock and it's fixed. Sorry for this.
The misleading thing is that the 2 instances were showing only after suspension.

**Comment by vanvugt** [comment]
> @3v1n0 @micheleg @vanvugt Given this issue has been raised several times and the default behavior is confusing newcomers, might it be a option to built a presence detection of some sort within Ubuntu Dock that it disables itself when the regular Dash to Dock extension is enabled?

Sorry for the delay. I've only just joined the project properly now. I don't think conflict detection is a priority because users can always break their machines in ways we can't imagine. It sounds like a reasonable request, but should be a separate issue to this one.

Duplicate of #1700
